js/view.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+function renderField(startX, startY, destinationX, destinationY, size, matr) {
```

Comment:
It's not recommended to pass more than 3-4 parameters in function.
Also some of this parameters are global variables.
Also it's better not to store data in global variables, use class or object for incapsulation and saving a game state.